kolibri/core/assets/src/views/nav-bar/index.vue

```diff
@@ -50,7 +50,7 @@
 
 <script>
 
-  const values = require('lodash.values');
+  const values = require('lodash/values');
```

Comment:
> You're requiring lodash, so webpack goes to lodash's folder in the modules, then you're specifying a specific subdirectory?

It should pick up the js file with that name.

e.g. this example points to _node_modules/lodash/values.js_

The idea here is that we don't need to individually add each lodash submodule to package.json. Doing it this way makes everything available at build time, but only includes what's used at load time.

> I'm getting buble build errors, saying these don't resolve to anything. Am I missing something?


that's weird - does the file I mentioned above exist in your project folder?